airlock/templates/update_files.html

```diff
@@ -0,0 +1,41 @@
+{# hide default button, as we are going to autoshow the modal anyway #}
+{% fragment as button %}
+{% endfragment %}
+<style>
+  div.filetype-radio div { display: hidden; }
+  div.filetype-radio label { display: hidden; }
+</style>
```

Comment:
I think everything above here can be removed; we don't need to hide the button, or set `custom_button=button` in the line below. We don't define any custom `button_text`, so the default modal button is hidden anyway.
And the formset that's used in this template doesn't have a filetype-radio, so there's no need to hide it.

Reply:
absolutely right, removed.

Reply:
it seems we do need `custom_button=button` & some other lines, otherwise we get an artefact on the parent page, as per this screenshot:

<img width="378" alt="Screenshot 2024-08-14 at 11 44 56" src="https://github.com/user-attachments/assets/eaf803c4-a53f-4ded-90eb-b524f1d14201">